### DISCUSSION TITLE
Show the version for each dependency listed (for a specific version as well)

### DISCUSSION THREAD

**Discussion by Alex-Sessler** [community/community#128435] [npm]
I came across this today while upgrading Angular from v11 to v12. Since v12 isn't the latest version, I am forced to do a lot of research to find the compatible versions for all of our dependencies.
The usual flow goes like this:
1. Upgrade the dependency (often times to a non-latest version)
2. Figure out which of my other dependencies depend on this
3. Go to the github repo of these dependencies, and figure out which released version matches the new version I just installed by going through the package.json tag by tag and figuring out which version is compatible

If you just showed the dependencies and peerDependencies for each version, I would not have to navigate back and forth between npm and github and could do it all while staying on the npm website.

**Comment by Eomm** [comment]
Totally agree.

I found myself learning releases date to calculate if a package could be worth trying it or not

This page could be so helpful to do this

<img width="1202" alt="image" src="https://user-images.githubusercontent.com/11404065/210590111-1932ad47-41a0-4dfc-8991-6d26a9c1f262.png">

**Comment by ThisIsMissEm** [comment]
Opened a related issue with regards to not displaying `peerDependencies`, which includes a mockup of the idea proposed here for showing versions / version ranges: https://github.com/orgs/community/discussions/158791
